packages/components/src/tooltip/test/index.tsx

```diff
@@ -101,10 +95,12 @@ describe( 'Tooltip', () => {
 		expect(
 			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
 		).not.toBeInTheDocument();
+
+		await cleanupTooltip( user );
 	} );
 
 	it( 'should respect custom delay prop when showing tooltip', async () => {
-		const user = userEvent.setup( { delay: TOOLTIP_DELAY } );
```

Comment:
This caused the test to exceed timeout after changing to the utility function. However, I don't think this is correctly used for the context we are testing here.